src/core/reference/src/runtime/reference/multiclass_nms.cpp

```diff
@@ -20,7 +20,189 @@ using namespace ngraph::runtime::reference;
 namespace ngraph {
 namespace runtime {
 namespace reference {
-namespace multiclass_nms_v8 {
+namespace multiclass_nms_impl {
+
+namespace {
+std::vector<float> get_floats(const std::shared_ptr<HostTensor>& input, const Shape& shape) {
```

Comment:
These get_xxx helpers are copied from evaluate_map.cpp. Can we put them in some common utility files? @ilyachur